### PR TITLE
[EXP-98-298] fix(test): more specific checkout ref

### PR DESCRIPTION
**Pull request by BobTheBuidler**

### CODE REVIEW

File: .github/workflows/tests.partners.on_pull_req.yaml [hunk 1]
```diff
@@ -38,8 +38,8 @@ jobs:
       - name: Check out repository code
         uses: actions/checkout@v2
```
Comment (mariuspod) [inline .github/workflows/tests.partners.on_pull_req.yaml:39]: we might bump this to `actions/checkout@v3`
https://github.com/actions/checkout/compare/releases/v2...main

File: .github/workflows/tests.partners.on_push.yaml [hunk 2]
```diff
@@ -34,6 +34,8 @@ jobs:
     steps:
       - name: Check out repository code
         uses: actions/checkout@v2
```
Comment (mariuspod) [inline .github/workflows/tests.partners.on_push.yaml:36]: we might bump this to `actions/checkout@v3`
https://github.com/actions/checkout/compare/releases/v2...main

**Review by mariuspod — APPROVED** [review]

File: .github/workflows/tests.partners.on_pull_req.yaml (hunk 1 above)
Comment (BobTheBuidler) [reply .github/workflows/tests.partners.on_pull_req.yaml]: bumped. tests running once more and then I'll merge.

File: .github/workflows/tests.partners.on_push.yaml (hunk 2 above)
Comment (BobTheBuidler) [reply .github/workflows/tests.partners.on_push.yaml]: bumped. tests running once more and then I'll merge.

**Pull request merged**